src/track.js

```diff
@@ -191,6 +206,12 @@ export default function extractTracks(file) {
                 });
             }));
 
+    case 'igc':
+        return readFile(file, 'text', isGzipped)
+            .then(textContents => new Promise((resolve) => {
+                resolve(extractIGCTracks(IGCParser.parse(textContents, {lenient: true})));
```

Comment:
Could you wrap the parsing in a `try / catch` and pass the caught exception to `reject()`? Want to make sure we don't fully error out when given a malformed IGC file

Reply:
I think it may be redundant (per step#10 in https://tc39.es/ecma262/#sec-promise-executor) but sure, why not - done.